web/src/app/modules/shared/components/smart/editor/editor.component.ts

```diff
@@ -58,6 +59,8 @@ export class EditorComponent extends AbstractViewComponent<EditorView> {
       this.pristineValue = view.config.value;
       this.options.readOnly = view.config.readOnly;
       this.options.language = view.config.language;
+      this.uri =
+        'file:text-' + Math.random().toString(36).substring(2, 15) + '.yaml';
```

Comment:
should this be in the constructor instead? Creating unnecessary churn in the object model can only hurt performance.

Reply:
Fixing this as part of #1396